packages/fxa-content-server/tests/server/routes/validate-email-domain.js

```diff
@@ -57,7 +57,17 @@ registerSuite('routes/validate-email-domain', {
           assert.isTrue(res.json.calledOnceWith({ result: 'MX' }));
         });
       },
-
+      'responds with result {result: "skip"} when there is a skip record}': () => {
+        const validateEmailDomainRoute = require('../../../server/lib/routes/validate-email-domain');
```

Comment:
@chenba here we have called the dunction from the validate-email-domain file rather in other tests we are doing the same via proxyrequiredns. Earlier this thing was ok since no config was passed but now it breaks. So we have to update  all the validateEmailDomainROutes to require from the file and while making the routes we have to pass config object there . Because in other tests if we see  `route.process` function is any and in the test file I wrote vscode intellisence say the `process ` function which I called.

Reply:
The fix for the existing tests are almost identical to what's done here.  Instead of the stub for the mocked config's `get` returning false, it'd return true.